airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -712,7 +709,141 @@ public void testResetCancelsRunningSync() throws Exception {
     assertEquals(JobStatus.CANCELLED, connectionSyncReadAfterReset.getStatus());
   }
 
-  // This test is disabled because it takes a couple minutes to run, as it is testing timeouts.
+  @Test
+  public void testSyncAfterUpgradeToPerStreamState(final TestInfo testInfo) throws Exception {
```

Comment:
Is someone else working on an acceptance test to test actual per-stream resets, i.e. sync a few streams, reset one of them, and verify that only that one stream is reset in the destination and that the state looks correct?

Reply:
Not exactly, I am currently working on an acceptance test for current reset behavior. I am happy to help out on that test afterwards.

Reply:
I have some room, I will work on those.